docs/database/database-metrics.md

```diff
@@ -35,6 +37,134 @@ and attributes but more may be added in the future.
 > until a transition plan to the (future) stable semantic conventions has been published.
 > Conventions include, but are not limited to, attributes, metric and span names, and unit of measure.
 
+## Database call
+
+### Metric: `db.client.call.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is [required][MetricRequired].
+
+When this metric is reported alongside a database operation span, the metric value SHOULD be the same as the database operation span duration.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 ]`.
+
+<!-- semconv metric.db.client.call.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `db.client.call.duration` | Histogram | `s` | Duration of database client calls. |
+<!-- endsemconv -->
+
+<!-- semconv metric.db.client.call.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`db.connection_string`](../attributes-registry/db.md) | string | The connection string used to connect to the database. It is recommended to remove embedded credentials. | `Server=(localdb)\v11.0;Integrated Security=true;` | Recommended |
```

Comment:
non-blocking: I'm questioning usefulness of this attribute in https://github.com/open-telemetry/semantic-conventions/issues/724, and have some concerns about others, but we can take it outside of this PR and potentially remove them from the metrics later on.

Reply:
Agreed. While I think connection string should be low-cardinality, I'm hard pressed to imagine a use case where it would be useful on a metric. Less is more, I'll remove it for now.